### DISCUSSION TITLE
Development Kickoff Meeting

### DISCUSSION THREAD

**Discussion by alexanderson1993** [Thorium-Sim/thorium-nova#160] [Q&A] [answered]
Let's schedule another meeting! The purpose of this meeting is to get interested in contributing code to Thorium Nova up to speed and figure out what needs to happen to get them started as contributors.

Here's the rough agenda.

- Talk about what features we want to build out first.
- Discuss the capabilities and API of the Thorium Nova engine.

Let's schedule it for sometime during the week of January 17 - 22. That hopefully gives enough time to get the word out and get prepared.

So, two questions:

- What day and time should we do the meeting?
- Is there anything else that should be added to the agenda?

Same rules as usual for scheduling. Anyone can propose a time, whichever proposal gets the most upvotes will be selected. We'll officially schedule the meeting on January 6th.

**Comment by astrolamb-gaming** [comment, marked as answer]
Tuesday, Wednesday, or Thursday starting at around 6:30 or 7:00 PM EST would be good for me. I believe that time has worked pretty well for our past meetings. Given a choice I think I'd prefer Wednesday myself.

**Reply by alexanderson1993** [reply]
Thank goodness somebody proposed a time! 😅

Wednesday, January 19th at 6:30pm EST sounds great to me. I doubt there will be any other proposals, so we'll plan on that. I'll send out the email tomorrow.

**Reply by 1VinceP** [reply]
That time works for me as well. How long do you expect the meeting to last?

**Reply by alexanderson1993** [reply]
Given that I have a free Zoom account, we're limited to 45 minutes, which is a good limit to have anyway. I don't want to take too much of people's afternoon/evening. (Although sometimes we re-enter the same Zoom meeting again which starts the clock over)
